app/views/admin/communities/edit_look_and_feel.haml

```diff
@@ -58,8 +58,8 @@
       .row
         .col-12
           - post_new_listing = "<i>#{t("homepage.index.post_new_listing")}</i>"
-          = form.label :custom_color2, t(".community_custom_color2", post_new_listing: post_new_listing).html_safe, :class => "input"
-          = render :partial => "layouts/info_text", :locals => { :text =>  t(".custom_color2_instructions_text", post_new_listing: post_new_listing).html_safe }
+          = form.label :custom_color2, t(".new_listing_button_custom_color").html_safe, :class => "input"
```

Comment:
At least in new top bar, we're using custom color #1 as the color for "Add listing" button: https://github.com/sharetribe/sharetribe/blob/b5a16211329d29fd921bd9fe3141bab3eaa6995e/client/app/components/sections/Topbar/Topbar.js#L324